components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -1014,7 +1015,7 @@ export class TypeORMWorkspaceDBImpl extends TransactionalDBImpl<WorkspaceDB> imp
         }
 
         const res = await query.getMany();
-        return res;
+        return res as PrebuiltWorkspaceWithWorkspace[];
```

Comment:
@svenefftinge Did you check if `res.workspace` is filled? I don't recall from the top of my head how/where `innerJoinAndMapOne` puts the workspace into the result shape.

Reply:
yes, I added a unit test